src/cachers/redis.js

```diff
@@ -114,6 +117,19 @@ class RedisCacher extends BaseCacher {
 		this.serializer = Serializers.resolve(this.opts.serializer);
 		this.serializer.init(this.broker);
 
+		// add interval for ping if set
+		if (typeof this.opts.pingInterval === "number") {
```

Comment:
Would be to skip the type checking. Just check `this.opts.pingInterval > 0` and in `setInterval` use `Number(this.opts.pingInterval). The reason is that if you try to configure it via environment variables (MOL_CACHER__OPTIONS__PING_INTERVAL=5000), the value will be `String`

Reply:
Good call.  I've made the proposed change and added a few more tests for it.